backend/apps/events/services/event_bus.py

```
@@ -80,7 +80,10 @@ def dispatch(cls, event: DomainEvent, sync: bool = False):
             # Process asynchronously (via Celery)
             from apps.events.tasks import process_event
 
+        try:
             process_event.delay(str(event.id))
+        except Exception:
+            pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)